pkg/cmd/roachprod/main.go

```diff
@@ -885,6 +885,47 @@ var getProvidersCmd = &cobra.Command{
 	},
 }
 
+var grafanaStartCmd = &cobra.Command{
+	Use:   `grafana-start <cluster> [<grafanaConfig>]`,
```

Comment:
Omit the `[<grafanaConfig>]`, cobra will render the config flag properly for you as part of `roachprod grafana-start -h`.